brave/brave-core
# Use brave search as a default search provider in tor window and update TorWindow NTP

Resolves https://github.com/brave/brave-browser/issues/20583

<img width="1259" alt="Screen Shot 2022-01-25 at 9 37 48 AM" src="https://user-images.githubusercontent.com/6786187/150888743-35bbeea5-5029-4da3-b5b3-33be9aba63a3.png">

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch Brave and open tor window
2. Check search provider is always Brave search
3. Check DDG graphic is not displayed
